packages/@sanity/components/src/dialogs/PanePopover.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable complexity */
+import React from 'react'
+import PropTypes from 'prop-types'
+import CheckCircleIcon from 'part:@sanity/base/circle-check-icon'
+import WarningIcon from 'part:@sanity/base/warning-icon'
+import ErrorIcon from 'part:@sanity/base/error-icon'
+import InfoIcon from 'part:@sanity/base/info-icon'
+import classNames from 'classnames'
+import styles from './styles/PanePopover.css'
+
+export default class PanePopover extends React.PureComponent {
+  static propTypes = {
+    children: PropTypes.node,
+    icon: PropTypes.oneOfType([PropTypes.string, PropTypes.node, PropTypes.bool]),
+    kind: PropTypes.oneOf(['info', 'warning', 'error', 'success', 'neutral']),
+    title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+    subtitle: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+    id: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired
+  }
+
+  static defaultProps = {
+    children: null,
+    icon: null,
+    kind: 'info',
+    title: 'Pane Popover',
+    subtitle: 'Popover content'
```

Comment:
This will make subtitle always be present - I think it should either not have a default subtitle or the check for the subtitle on render can be removed

Reply:
Yes, you're right!